MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java

```diff
@@ -337,7 +337,7 @@ private void drawShadow(Canvas canvas) {
      */
     private void update(boolean force, boolean async) {
         View view = mViewRef.get();
-        if (view != null) {
```

Comment:
`CompatElevationUpdateRunnable` is being run when elevation is 0 and it crashes if elevation before was >0 :) 
But I must say I had hard times going through this code, so my fix maybe not the best possible solution 🙄